script/Arcueid/Day 14/14_00_ARC14_1.txt

```diff
@@ -990,7 +990,7 @@ For her to... I―――
 [sha:2b715fd0a1c115d6f3ea988021e7d865bc3a8c47]{
 -- Page 62, Offset 36938.
 -- 「……優しいね、志貴。
-"...You really are kind, aren't you Shiki.
+"...You really are kind, aren't you, Shiki.
```

Comment:
Could take or leave this one

Reply:
I like lots of pauses in Aoko speech, it feels very warm